javaslang-benchmark/src/test/java/javaslang/collection/ArrayBenchmark.java

```diff
@@ -77,43 +127,51 @@ public void tearDown() {
         }
 
         @Benchmark
-        public void java_mutable(Initialized state) {
+        public Object java_mutable(Initialized state) {
```

Comment:
Good question.
I followed the same logic as in choosing whether I want a test to
```java
public void test () throws Exception
```
or
```java
public void test () throws SomeVerySpecificExceptionThatNobodyCaresAboutException
```

i.e. the return type of a benchmark is meant as a dummy, to avoid dead code elimination, the returned element won't be used.
It's easier to ignore it as a user, if it's always an `Object`.
(potential boxing (e.g. in iterate) shouldn't be a real problem, as the inside operations outweigh the cost).